deps/ncrypto/ncrypto.cc

```diff
@@ -1998,6 +2010,32 @@ DataPointer EVPKeyPointer::rawPublicKey() const {
   return {};
 }
 
+#if OPENSSL_VERSION_MAJOR >= 3 && OPENSSL_VERSION_MINOR >= 5
+DataPointer EVPKeyPointer::rawSeed() const {
+  if (!pkey_) return {};
+  switch (id()) {
+    case EVP_PKEY_ML_DSA_44:
+    case EVP_PKEY_ML_DSA_65:
+    case EVP_PKEY_ML_DSA_87:
+      break;
+    default:
+      unreachable();
+  }
+
+  size_t seed_len = 32;
+  if (auto data = DataPointer::Alloc(seed_len)) {
+    const Buffer<unsigned char> buf = data;
+    size_t len = data.size();
+    // TODO(@panva): use OSSL_PKEY_PARAM_ML_DSA_SEED instead of "seed"
+    if (EVP_PKEY_get_octet_string_param(
+            get(), "seed", buf.data, len, &seed_len) != 1)
```

Comment:
Help would be appreciated, maybe I misunderstood and there is no OSSL_PKEY_PARAM_ML_DSA_SEED variable to use instead of "seed" verbatim?